picolibc.h.in

```diff
@@ -142,9 +88,6 @@
 /* Support _init() and _fini() functions */
 #cmakedefine _HAVE_INIT_FINI
 
-/* Compiler has long double type */
-#cmakedefine _HAVE_LONG_DOUBLE
```

Comment:
Could also use 
```
#ifdef __SIZEOF_LONG_DOUBLE__
#define _HAVE_LONG_DOUBLE
#endif
```
To avoid using the compiler-provided macro directly everywhere else?

Reply:
Yeah, I considered that, but decided that getting rid of a symbol was worth the small reduction in clarity. Perhaps I'm wrong though?

Reply:
you're right -- using `_HAVE_LONG_DOUBLE` makes the code a lot clearer. I've switched back.